### DISCUSSION TITLE
2.3.0-RC

### DISCUSSION THREAD

**Discussion by alganet** [Respect/Validation#1409] [Announcements]
## 2.3-RC

Versioning Changes:

 - Dropped support for PHP 7.4.
 - Updated dev dependencies

Deprecations:

 - Symfony façade validators are no longer supported and were
   removed.

Fixes:

 - `KeySet` now reports which extra keys are causing the rule to fail.

Changes:

 - You can no longer wrap `KeySet` in `Not`.
 - `Phone` now uses `giggsey/libphonenumber-for-php`, this package needs
   to be installed if you want to use this validator.
 - `Phone` now supports the parameter `$countryCode` to validate phones
   of a specific country.

<hr /><em>This discussion was created from the release <a href='https://github.com/Respect/Validation/releases/tag/2.3.0-RC'>2.3.0-RC</a>.</em>

**Comment by henriquemoody** [comment]
I'm closing this as we've already released version 2.3. 

Again, I'm sorry I missed so many discussions before releasing, @alganet
